python/triton/compiler.py

```diff
@@ -891,6 +891,8 @@ def ttir_to_ttgir(mod, num_warps, num_stages, compute_capability):
     # The combine pass converts blocked layout to mma layout
     # for dot ops so that pipeline can get shared memory swizzled correctly.
     pm.add_triton_gpu_combine_pass(compute_capability)
+    # The update_mma_for_volta pass helps to compute some information for MMA encoding specifically for MMAv1
+    pm.add_triton_gpu_update_mma_for_volta_pass()
```

Comment:
Can we only add the pass if compute_capability==70 ?

Reply:
Done.